srcpkgs/volta/template

```diff
@@ -1,21 +1,17 @@
 # Template file for 'volta'
 pkgname=volta
-version=0.9.3
+version=0.9.99
 revision=1
 archs="x86_64 i686" # Due to volta pulling pre-built binaries later. Evil.
-build_helper="rust"
-hostmakedepends="cargo pkg-config"
+build_style="cargo"
+hostmakedepends="pkg-config"
 makedepends="libressl-devel"
 short_desc="JavaScript tool version sync helper"
 maintainer="Alex Lohr <alex.lohr@logmein.com>"
 license="BSD-2-Clause"
 homepage="https://volta.sh/"
 distfiles="https://github.com/volta-cli/volta/archive/v${version}.tar.gz"
-checksum=3777585d5ebd24f9f53ac5d41c378c471211ded0d8e0e126110648fefbd0d316
-
-do_build() {
-	cargo build --release --target ${RUST_TARGET}
-}
+checksum=7485db7471f44de574523152ed14bbda549954794e6c2fd920a9d80b2ebcdb9b
 
 do_install() {
```

Comment:
Can you replace do install as well, and only use a post install for the license?

Reply:
Thanks for the suggestion. I'll do it at once.